docs/design/extensions-contrib/dropwizard.md

```diff
@@ -629,6 +629,38 @@ Latest default metrics mapping can be found [here] (https://github.com/apache/dr
       "priority"
     ],
     "type": "gauge"
+  },
+  "jetty/numOpenConnections": {
+    "dimensions": [],
+    "type": "gauge"
+  },
+  "jetty/threadPoolNumTotalThreads": {
```

Comment:
I think it would be more clear to name the metric as `jetty/threadPool/total`. So do other names.

Reply:
Thanks! Addressed.